css/index.css

```diff
@@ -142,6 +142,12 @@ body {
   margin: 0 auto;
   width: 100%;
```

Comment:
make sure you try adding padding and border and you could try to do vertical align at the baseline.